src/react-components/src/components/02-molecules/SearchBar/SearchBar.tsx

```diff
@@ -18,6 +22,7 @@ export interface SearchBarProps {
   placeholderText?: string;
   textFieldAriaLabel?: string;
   textFieldAttributes?: {};
+  helperVariant?: "ResearchNow";
```

Comment:
This says that `helperVariant` can only be of type "ResearchNow" (just this value). This should be `helperVariant?: string;` and the default prop value for this can then be "ResearchNow". Otherwise, any other value won't be accepted. I don't _think_ this is what you meant if this component is meant to be generic with any value for `helperVariant`.
![Screen Shot 2020-01-08 at 4 55 04 PM](https://user-images.githubusercontent.com/1280564/72019411-a3bec500-3237-11ea-9b64-073b6bf950ef.png)

Reply:
It has to be "ResearchNow" - there aren't other variants and there aren't any in the plans

Reply:
I see. If this should only be used in RN, then I don't think the Design System is the right place for it. I meant my comment to be more of a Typescript issue rather than DS philosophy.

Reply:
Yeah, this ends up kinda dovetailing into DS philosophy - there's a world in which DS (at least, this storybook) contains everything that exists, so that we can test everything that ever changes to DS.  I don't know what the pros/cons of that world are, I'm about to push something that might be even more tricky re: that so uh... stay tuned. 

Re typescript:  knowing this rationale, do you think this is ok as is (Maybe with a comment or a test that makes this clear?) or do you think it should still accept a generic string?

Reply:
I think it should accept a generic string and make the value `"ResearchNow"` a default prop. But, if you leave it as is and add a comment (and maybe a todo) about why this is the case, that'd be just as helpful.